GOVERNANCE.MD

```diff
@@ -1,6 +1,92 @@
 # Community Committee Governance
 
-The Community Committee governance is currently under pull request as the Community Committee Charter.
+Section 1. Guiding Principle.
+
+The Community Committee will operate transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to the entire ecosystem.
+
+Section 2. Evolution of Node.js Foundation Governance.
```

Comment:
One thing we discussed in a recent TSC meeting about the CC, was having someone from each committee regularly attend the other's meeting. Whether those people would be observers or full members was uncertain, but I think that would be an important step in making sure the two committee's get along :)